kolibri/core/errorreports/middleware.py

```diff
@@ -1,12 +1,37 @@
 import logging
 import traceback
+from sys import version_info
 
+import pkg_resources
```

Comment:
We should avoid using pkg_resources if you at all possible, it is a deprecated API and also causes incredibly long import times.

See this stackoverflow answer for an alternative that will work here: https://stackoverflow.com/questions/49923671/are-there-any-function-replacement-for-pip-get-installed-distributions-in-pip#49923876